src/loaders/google_map_loader.js

```diff
@@ -1,87 +1,89 @@
 const BASE_URL = 'https://maps';
 const DEFAULT_URL = `${BASE_URL}.googleapis.com`;
-const API_PATH = '/maps/api/js?callback=_$_google_map_initialize_$_';
+const API_PATH = '/maps/api/js?callback=googleMapsapiLoadedPromise';
```

Comment:
Can we rename to `googleMapsAPILoadedPromise`

Reply:
Fixed